tools/SampleWinGetUWPCaller/AppInstallerCaller/Package.appxmanifest

```diff
@@ -26,6 +26,7 @@
   </Applications>
   <Capabilities>
     <Capability Name="internetClient" />
+    <rescap:Capability Name="packageQuery" />
```

Comment:
>packageQuery [](http://example.com/codeflow?start=29&length=12)

packageManagement should already include packageQuery in semantics, did you find places that only work with packageQuery but not packageManagement? If yes, we may need to fix those.

Reply:
Ok, it looks like the code is doing that and I was just seeing the failure to directly find `packageQuery` while debugging.  I can revert this and update the code to not log a failure here.